mimesniff/mime-types/resources/mime-charset.py

```diff
@@ -0,0 +1,3 @@
+def main(request, response):
+    response.headers.set("Content-Type", request.GET.first("type"));
+    response.content = "<meta charset=utf-8>\n<script>document.write(document.characterSet)</script>"
```

Comment:
FWIW you can also write this as

```
def  main(request, response):
    return ([("Content-Type", request.GET.first("type"))],
            "<meta charset=utf-8>\n<script>document.write(document.characterSet)</script>")

Reply:
Is that better?

Reply:
Up to you; it was just in case you didn't know.